Makefile

```
@@ -1,190 +1,310 @@
+	$(call docker_build,acra-tools)
+	$(call docker_build,acra-webconfig)
+	$(call docker_build,acra-authmanager)
+	$(DOCKER_BIN) image prune --force -a \
+		--filter label=com.cossacklabs.product.name="$(APP_NAME)" \
+		--filter label=com.cossacklabs.docker.container.type="build"
+
+## Docker : tag and push image to remote registry
+docker-push:
+	$(call docker_push,acra-server)
+	$(call docker_push,acra-connector)
+	$(call docker_push,acra-translator)
+	$(call docker_push,acra-keymaker)
+	$(call docker_push,acra-tools)
+	$(call docker_push,acra-webconfig)
+	$(call docker_push,acra-authmanager)
+
+## Docker : remove stopped containers and dangling images
+docker-clean:
+	$(DOCKER_BIN) container prune --force \
+		--filter label=com.cossacklabs.product.name="$(APP_NAME)"
+	$(DOCKER_BIN) image prune --force -a \
+		--filter label=com.cossacklabs.product.name="$(APP_NAME)"
+
+## Docker : alias for 'docker-build' target (DEPRECATED)
+docker: docker-build
+
+##---- Packages ----------------------------------------------------------------
+
+## Package : build deb/rpm depending on the current OS
+pkg: build
+ifeq ($(OS_NAME),unknown)
+	$(error OS is not detected)
+endif
+ifeq ($(OS_CODENAME),unknown)
+	$(error OS codename is not detected)
+endif
+ifeq ($(OS_VERSION_MAJOR),unknown)
+	$(error OS version is not detected)
+endif
+ifeq ($(OS_NAME),$(filter $(OS_NAME),Debian Ubuntu))
+# acra_0.85.0+stretch_amd64.deb
+# acra_0.84.0-53-gd90b699+stretch_amd64.deb
+	$(eval PKG_TYPE := deb)
+	$(eval PKG_VERSION := $(APP_VERSION)+$(OS_CODENAME))
+	$(eval PKG_NAME := $(APP_NAME)_$(PKG_VERSION)_$(OS_ARCH_DEBIAN).deb)
+	$(eval PKG_TYPE_SPECIFIC_ARGS := --deb-priority optional \
+		--architecture $(OS_ARCH_DEBIAN))
+else ifeq ($(OS_NAME),$(filter $(OS_NAME),RedHatEnterpriseServer CentOS))
+# acra-0.85.0.centos7.x86_64.rpm
+# acra-0.84.0_54_g41001c5.centos7.x86_64.rpm
+	$(eval PKG_TYPE := rpm)
+	$(eval PKG_VERSION := $(shell printf "$(APP_VERSION)"|sed s/-/_/g))
+	$(eval PKG_NAME := $(APP_NAME)-$(PKG_VERSION).$(OS_SHORTNAME)$(OS_VERSION_MAJOR).$(OS_ARCH_RHEL).rpm)
+	$(eval PKG_TYPE_SPECIFIC_ARGS := --rpm-summary $(APP_LONG_DESCRIPTION) \
+		--architecture $(OS_ARCH_RHEL))
+else
+	$(error packaging for OS $(OS_NAME) is not supported yet)
+endif
+	@mkdir -p '$(BUILD_DIR_ABS)/$(PKG_TYPE)'
+	@mkdir -p '$(BUILD_DIR_ABS)/$(PKG_TYPE).struct/bin'
+	@for p in $(PKG_COMPONENTS); do \
+		cp "$(BUILD_DIR_ABS)/bin/acra-$$p" '$(BUILD_DIR_ABS)/$(PKG_TYPE).struct/bin/'; \
+	done
+	@fpm \
+		--input-type dir \
+		--output-type $(PKG_TYPE) \
+		--name $(APP_NAME) \
+		--license '$(APP_LICENSE_NAME)' \
+		--url '$(APP_VENDOR_URL)' \
+		--description $(APP_SHORT_DESCRIPTION) \
+		--vendor '$(APP_VENDOR_NAME)' \
+		--maintainer '$(shell printf "$(APP_VENDOR_NAME) <$(APP_VENDOR_EMAIL)>")' \
+		--package '$(BUILD_DIR_ABS)/$(PKG_TYPE)/$(PKG_NAME)' \
+		--version '$(PKG_VERSION)' \
+		--category security \
+		--depends openssl \
+		--depends libthemis \
+		--conflicts acra \
```

I believe this should be
```suggestion
		--conflicts acra-ee \
```

---

Sure. Thanks!